networking/src/ModelHelper.ts

```diff
@@ -100,20 +100,18 @@ export abstract class ModelHelper {
         }
 
         const correctAnswer = this.getCorrectAnswer(question)
+        const currentQuestion = gameSession?.questions[gameSession?.currentQuestionIndex ?? 0]
+        let originalScore = gameSession?.teams?.find(teamElement => teamElement.id === team.id)?.score ?? 0 
```

Comment:
Why not just use the `team` object that's passed in?

Reply:
resolved, using team.score in place of originalScore